ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-community-4.12__okd.yaml

```diff
@@ -53,7 +53,13 @@ tests:
   container:
     from: src
 - as: aws-e2e-operator
-  skip_if_only_changed: ^(?:docs|\.github)/|\.md$|^(?:\.gitignore|OWNERS|PROJECT|LICENSE)$
+  cron: 0 0 * * *
+  reporter_config:
```

Comment:
`reporter_config` goes in the job definition.
https://docs.ci.openshift.org/docs/how-tos/notification/
> We can set up Slack alerts for job results by defining reporter_config in the job definition.

>Note that we have to modify the job yaml directly even if the job is generated from a ci-operator’s config and regenerating the jobs does not change the existing reporter_config.

Reply:
Thanks for pointing that out, I moved it to the job